docs/default_csi_plugin.md

```diff
@@ -0,0 +1,80 @@
+# MicroShift Default CSI Plugin
+
+MicroShift enables dynamic storage provisioning out of the box with the ODF-LVM CSI plugin. This plugin is downstream
```

Comment:
It's implied in the name, but you could be explicit here and say that ODF-LVM provides a separate logical volume for each PersistentVolumeClaim.